hpedockerplugin/volume_manager.py

```diff
@@ -1973,6 +1973,7 @@ def _decrypt_password(self, src_bknd, trgt_bknd, backend_name):
         try:
             passphrase = self._etcd.get_backend_key(backend_name)
         except Exception as ex:
+            LOG.error('Exception occurred %s ' % ex)
```

Comment:
As I understand it, if the backend key is not found with ETCD then that's the case of plain credentials. At best, I think this statement should be logged as "INFO" and not "ERROR".